feat(rules): Add SQL Generator function (#2961)
emqtt/emq-docs-cn: en_US/data-integration/rule-get-started.md

This tutorial assumes the message payload is JSON. If the payload is formatted in some other way, you can convert the data type, for example, with the [Schema Registry](./schema-registry.md).

EMQX has embedded rich SQL statement samples to help you get started. You can click the **SQL Examples** button under the **SQL Editor** to explore. For more details about the SQL syntax and usages, see [SQL Syntax](./rule-sql-syntax.md).

:::

<img src="./assets/rules/create-rules.png" alt="image-20230417211146211" style="zoom:50%;" />

### SQL Generator

::: tip Note

> The SQL Generator feature is available only in EMQX editions covered by a Commercial License.

Reviewer: ```suggestion
The SQL Generator feature is available only when EMQX is configured with a production license key.
```

> :::

Reviewer: sorry, this is actually open to everybody, no such limitation.
